typings/glob-to-regexp/index.d.ts

```diff
@@ -1,7 +1,9 @@
 declare module "glob-to-regexp" {
```

Comment:
Do we need to delcare module then at all? https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-function-d-ts.html looks like we should be able to declare a function and export it

Reply:
module declaration is needed if `import ... from "module"` syntax should stay, cf. https://github.com/Microsoft/TypeScript/issues/5073.

Reply:
if we use `import x = require('module')` then it should not be necessary

Reply:
well, it's still required.

> [ts]
Could not find a declaration file for module 'glob-to-regexp'. '/Users/tugarev/Projects/monaco-languageclient/node_modules/glob-to-regexp/index.js' implicitly has an 'any' type.
  Try `npm install @types/glob-to-regexp` if it exists or add a new declaration (.d.ts) file containing `declare module 'glob-to-regexp';`

Reply:
I double-checked that, it's required if types should be imported, right? module-function.d.ts seems to be misleading.